### DISCUSSION TITLE
Access the public domain via Jellyfin

### DISCUSSION THREAD

**Discussion by n-buck** [jellyfin/jellyfin#10304] [General]
Since a lot of intresting media can be found in the public domain, I propose to have the possibillty to share such media with other users.
This would also allow users, that cannot contribute with code or money, to contribute to the success of the jellyfin-project.
I also have a catchy name for this feature: Jellyfin - Ocean

A draft of how this could be achieved and rolled out:
* Allowing a public (or well-known) user, to access media marked by the server owner combined with a database of registered servers.
* Adding a manifest of the servers content, to provide an overview of what to expect when accessing it.
* Providing a summary of available available media directly. By selecting a media, the user can choose from which server it shall be streamed.

To be considered:
The public domain is not global, which makes this much more complicated.
I.E. One might be allowed to share a movie in America publicly, but  to download / stream it might be illegial in certan EU-countries.
-->How can a jellyfin-admin be protected from evt. law-sutes.

**Comment by cvium** [comment]
This is not something we would be willing to implement nor endorse.
